lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -360,6 +364,19 @@ sub _show {
     my ($self, $job) = @_;
     return $self->reply->not_found unless $job;
 
+    my $modules = read_test_modules($job);
```

Comment:
Yet another place where we read the test modules (which isn't that efficient by the way)? In my review for the other PR I've mentioned that you can simply use the columns of the jobs table (e.g. `passed_module_count`). This would also avoid the following loop.

Reply:
Done